packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -504,7 +505,7 @@ export const FIREHOSE_CIDR_BLOCKS: { [region: string]: string } = {
   'ap-northeast-2': '13.209.1.64',
   'ap-northeast-3': '13.208.177.192',
   'ap-south-1': '13.232.67.32',
-  'ap-south-2': '18.60.192.128',
+  'ap-south-2': '18.60.192.192',
```

Comment:
Is this on purpose?

Reply:
Nice catch, when looked up the doc provided it does have Kinesis Data Firehose using one CIDR block for each available Region in different scenarios. Fixed them with the required values. Thank you!